Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -533,6 +620,9 @@ def main() -> None:
             email_address = demisto.args()["email_address"]
             result = add_risky_user_label(reco_client, email_address)
             return_results(result)
+        elif command == "reco-get-assets-user-has-access-to":
+            result = get_assets_by_owner(reco_client, demisto.args()["asset_owner"], demisto.args()["only_sensitive"])
```

Comment:
The `asset_owner` argument is not in the yml file. Did you mean `email_address` ?